docs/knative-eventing-mesh/08-01-configure-kafka-channel.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Configure Kafka Channel
+type: Tutorials
+---
+
+Instead of the default Channel implementation, you can use the Knative-compatible [Kafka Channel](https://github.com/kyma-incubator/knative-kafka). To ensure Kafka works properly, you must:
+
+* Install the `knative-eventing-kafka`  component to deploy the Kafka controller
+* Set up a Kafka cluster using Azure Event Hubs
```

Comment:
If Kyma is already installed with `knative-eventing-kafka` component, creating the custer and the secret would be too late. The kafka channel controller will not start (and thus fail the installation) If I remember correctly.